app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if category.children.any? %>

```

Comment:
Vì sao check any mà không check nil? hay blank?

Reply:
Dạ, tại vì nil? và blank? sẽ duyệt qua toàn bộ bản ghi, any? dùng select count(*) và dừng lại khi tìm thấy bản ghi đầu tiên, giúp tiêt kiệm tài nguyên ạ.